src/datreant/core/util.py

```diff
@@ -17,3 +20,35 @@ def makedirs(path):
             pass
         else:
             raise
+
+
+def touch_me(path):
+    Path(path).touch()
+
+
+def fullpath(path):
+    if not path:
+        return None
+    if not isinstance(path, six.string_types):
+        path = str(path)
+    path = os.path.expanduser(path)
+    if not os.path.isabs(path):
+        path = os.path.abspath(path)
+    return path
+
+
+def isfullpath(path):
+    return path == fullpath(path)
+
+
+def path_leaf(path):
+    return os.path.basename(os.path.normpath(path))
+
+
+def relpath(path):
+    """Returns *path* on Windows, and relative path elsewhere."""
```

Comment:
Np..

https://bugs.python.org/issue7195 
http://stackoverflow.com/questions/4950940/os-path-relpath-not-returning-a-relative-path-when-using-drive-as-start-point

relpath works on windows if not one path is on D:\ and one an E:\ ..between this two there is no relpath. On my Windows system the python env is on D:\ the system on C:\ so i add a tmpdir for the test's 

Also within the windows tmp there a permission issues on WIN10